src/main/java/com/google/cloud/tools/appengine/api/devserver/DefaultRunConfiguration.java

```diff
@@ -172,7 +177,7 @@ public void setPythonStartupArgs(String pythonStartupArgs) {
   }
 
   public void setJvmFlags(List<String> jvmFlags) {
-    this.jvmFlags = jvmFlags;
+    this.jvmFlags = jvmFlags != null ? ImmutableList.copyOf(jvmFlags) : null;
```

Comment:
For the first question - yeah probably. I only touched this field unrelated to this PR because it's in same class. We can deal with the rest of the cases in a different PR.

No I don't think it's expected that they change it. It's just to protect against them unknowingly (or knowingly even but for other reasons) changing it and getting unexpected side effects. Maybe I misunderstood your point.